src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx

```diff
@@ -305,8 +315,31 @@ const ManageReputationDialogForm = ({
               />
             )}
           </div>
+          {!isSmiteAction && (
+            <div className={styles.warningContainer}>
+              <p className={styles.warningTitle}>
+                <FormattedMessage {...MSG.warningTitle} />
+              </p>
+              <p className={styles.warningText}>
+                <FormattedMessage
+                  {...MSG.warningText}
+                  values={{
+                    a: (chunks) => (
+                      <ExternalLink
+                        href={REPUTATION_LEARN_MORE}
+                        className={styles.link}
+                      >
+                        {chunks}
+                      </ExternalLink>
+                    ),
+                  }}
+                />
+              </p>
+            </div>
+          )}
         </div>
       </DialogSection>
+      {!isSmiteAction && <hr className={styles.divider} />}
```

Comment:
Not a big fan of `hr` elements, but you used it correctly in this instance, so I can't even nit pick it :P